review-me/review-me.js

```diff
@@ -0,0 +1,31 @@
+// hello.js
+
+var message = "Hello, world!"
+console.log(MESSAGE);
+
+function printmessage(msg) {
+    if(msg != null){
+        console.log(msg)
+    }
+    else{
+        console.log("Nothing to print")
+    }
+}
+
+printmessage(message)
+
+eval("console.log('Eval should not be used!')")
+
+// Unused variable
```

Comment:
19  :  unused  is  declared  but  never  used  Code  smell  for  maintain ability